qiime2/sdk/tests/test_usage.py

```diff
@@ -359,4 +359,5 @@ def test_merge_metadata_one_input(self):
 class TestScopeRecord(TestCaseUsage):
     def test_invalid_assert_has_line_matching(self):
         with self.assertRaisesRegex(TypeError, 'should be a `callable`'):
-            usage.ScopeRecord('foo', assert_has_line_matching='spleen')
+            usage.ScopeRecord('foo', 'value', 'source',
+                              assert_has_line_matching='spleen')
```

Comment:
What is 'spleen'?

Reply:
That's a pretty deep question!

In this case - nothing, just a placeholder value to appease the constructor.